Pod/Classes/DMP.swift

```diff
@@ -364,6 +364,49 @@ open class DMP:NSObject{
             }.resume()
         }
     }
+
+    /**
+    Send an HTTP or HTTPs request using the supplied URL pattern.
+    This pattern can contain two replacement macros, {deviceid} and {deviceidtype},
+    which will be replaced before performing the HTTP(s) call.
+    */
+    
+    open class func sendRequest(urlPattern:String,_ completion:@escaping (_ result: Result<LotameProfile>)->Void) {
+        guard sharedManager.isInitialized else{
```

Comment:
move this two guards within the async block for consistency with the sendBehaviorData method